custody-fireblocks/src/main/kotlin/com/stablecoin/custody/fireblocks/domain/allocation/FundAllocationService.kt

```
@@ -0,0 +1,267 @@
+                fundAllocationRepository.save(released)
+
+                auditLogRepository.save(
+                    AuditLog.create(
+                        operation = AuditOperation.ALLOCATION_RELEASED_BY_CLEANUP,
+                        actor = "system",
+                        resourceId = released.id.toString(),
+                        status = AuditStatus.SUCCESS,
+                        details = mapOf("allocationId" to allocation.allocationId),
+                    ),
+                )
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)